### DISCUSSION TITLE
Run above and Run Below buttons not found in Jupyter in MacOS

### DISCUSSION THREAD

**Discussion by tdevinda** [microsoft/vscode-jupyter#9836] [Questions and Answers] [answered]
I have two buttons in each cell to run all the cells above or run all the cells below in my Windows VS code notebooks. But I cant enable that in macOS. 
<img width="351" alt="image" src="https://user-images.githubusercontent.com/4415594/165893137-56104b47-ec0f-4a23-ac48-63a2825787b0.png">
This is the mac's button list. I checked in the settings and it does have the run all cells listed here;
<img width="718" alt="image" src="https://user-images.githubusercontent.com/4415594/165893641-08e562ce-da22-4a91-a98d-8f72e3cfff8b.png">

But buttons do not appear. expected output is here;
https://github.com/microsoft/vscode-jupyter/issues/5823#issuecomment-841978676

any settings I am missing?

**Comment by IanMatthewHuff** [comment, marked as answer]
My guess would be that you have turned on consolidated run button. That's not jupyter specific, but instead a general notebook setting.

With the setting on run above and run below are moved into a dropdown on the run button, which I think I see in your picture. If you turn that off, the buttons will be back on the toolbar. 

<img width="1411" alt="image" src="https://user-images.githubusercontent.com/812783/166063227-89e73bfc-fec5-4d25-b699-5f9f3006f106.png">
<img width="1401" alt="image" src="https://user-images.githubusercontent.com/812783/166063295-ca6b68b4-c848-4381-96fd-38e396272326.png">

**Reply by tdevinda** [reply]
Solved. Yes. this is the issue. Thanks a lot
